### ISSUE TITLE
Fine tuning network

### ISSUE THREAD

**Issue by rayjs** [apache/incubator-mxnet#10077]
Hi,

I have a pretrained network and I am fine tuning it. I want to fix some of the weights of the network and modify the rest. I can specify the argument names that are to remain fixed via `fixed_param_names` (see below) but the auxiliary parameters(such as in batch norm) cannot be provided as arguments in `fixed_param_names`. Is it possible to fix the auxilary parameters? Please help.
 ```
class mxnet.module.Module(symbol, data_names=('data', ), label_names=('softmax_label', ), logger=, context=cpu(0), work_load_list=None, fixed_param_names=None, state_names=None, group2ctxs=None, compression_params=None)
```

**Comment by nswamy** [comment]
We manage user-questions on https://discuss.mxnet.io/, please post your question there. 
I am closing this issue.
